pkg/controller/datavolume-controller.go

```diff
@@ -455,25 +459,29 @@ func (r *DatavolumeReconciler) Reconcile(_ context.Context, req reconcile.Reques
 		}
 	}
 
-	if selectedCloneStrategy == SmartClone || selectedCloneStrategy == CsiClone {
-		if selectedCloneStrategy == CsiClone {
-			if pvc.Status.Phase == corev1.ClaimBound {
-				if err := r.setCloneOfOnPvc(pvc); err != nil {
-					return reconcile.Result{}, err
-				}
-			} else if pvc.Status.Phase == corev1.ClaimPending {
-				return reconcile.Result{}, r.updateCloneStatusPhase(cdiv1.CSICloneInProgress, datavolume, pvc, selectedCloneStrategy)
-			} else if pvc.Status.Phase == corev1.ClaimLost {
-				return reconcile.Result{},
-					r.updateDataVolumeStatusPhaseWithEvent(cdiv1.Failed, datavolume, pvc, selectedCloneStrategy,
-						DataVolumeEvent{
-							eventType: corev1.EventTypeWarning,
-							reason:    ErrClaimLost,
-							message:   fmt.Sprintf(MessageErrClaimLost, pvc.Name),
-						})
-			}
+	switch selectedCloneStrategy {
+	case HostAssistedClone:
+		if err := r.ensureExtendedToken(pvc); err != nil {
```

Comment:
Does this address the case where we start an import, then immediately start a clone of that import, and the import takes longer than the token is valid, so when we finally get to the clone part, it fails due to an expired token?

Reply:
Yeah that's basically the whole purpose of this PR.  The once the extended token is added to the target PVC (right after it's created), we've got 10 years to start the clone source pod

Reply:
So why only one host assisted clone? Doesn't the same issue apply to csi/smart clone?

Reply:
Object transfer is created at the start of the operation after the short token is validated.